data/presets/presets/amenity/kindergarten.json

```diff
@@ -30,7 +30,7 @@
         "point"
     ],
     "terms": [
-        "kindergarden",
```

Comment:
I'm a Brit and I've never seen the so called Anglicised version, I've re-added it as another term given having common typos here seems to be a good thing.

Reply:
Sounds fine.